src/app/common/templates/job-body.html

```diff
@@ -1,6 +1,13 @@
 <div class="job-body-heading">
     <div>
-        <div class="job-title">{{ job.attributes.name }}</div>
+        <div class="job-title">
+            <div ng-if="!(job.attributes.name_translation.from !== job.attributes.name_translation.to && dt.job_item)">
+                {{ job.attributes.name }}
+            </div>
+            <div ng-if="job.attributes.name_translation.from !== job.attributes.name_translation.to && dt.job_item">
```

Comment:
Again I'm not an Angular developer (sorry), but can't we extract the `ng-if` line to a named method to make it clear what being checked (without having to "parse" the line yourself)?